src/main/resources/assets/serverutilities/lang/zh_CN.lang

```diff
@@ -1,301 +1,514 @@
+# Misc
+gamerules=游戏规则
+serverutilities.lang.timer.shutdown=服务器将在%s后关闭
+serverutilities.lang.killed_entities=已杀死 %d 个类型为“%s”的实体
+serverutilities.lang.kickme=你踢出了自己！
+serverutilities.lang.upload_crash=上传崩溃报告
+serverutilities.lang.uploaded_crash=崩溃报告已上传！ %s
+serverutilities.lang.nickname_changed=昵称修改为“%s”
+serverutilities.lang.nickname_reset=昵称重置
+serverutilities.lang.unmuted=%s已经被取消禁言
+serverutilities.lang.muted=%s已经被禁言
+serverutilities.lang.fly_on=为%s开启飞行
+serverutilities.lang.fly_off=为%s关闭飞行
+serverutilities.lang.god_on=为%s开启无敌模式
+serverutilities.lang.god_off=为%s关闭无敌模式
+serverutilities.lang.heal=为%s治疗
+serverutilities.lang.permissions_dumped=%s权限已成功转储到文件路径：%s.
+serverutilities.disabled.config=在配置文件中禁用
+serverutilities.disabled.server=被服务器禁用
+serverutilities.navigator.button=显示服务器实用工具认领区块
+serverutilities.unsaved_changes.title=未保存的更改
+serverutilities.unsaved_changes=您有未保存的更改。是否要保存它们?
+serverutilities.command_overview=指令概览
+
+serverutilities.stat.time_afk=挂机时长
+serverutilities.stat.time_afk_percent=挂机时间占比
+serverutilities.stat.time_active=活跃时长（非挂机）
+serverutilities.stat.dph=每小时死亡次数
+serverutilities.stat.last_seen=最后在线时间
+
+wip=开发中！
+deprecated=此方法已弃用，请使用%s
+example=Example:
+click_here=[点击此处]
+feature_disabled=此功能已禁用！
+feature_disabled_server=此功能已被服务器禁用！
+error=错误：%s
+owner=所有者：%s
+delete_item=您是否要删除'%s'?
+server_forced=服务器强制修改为'%s'
+commands=指令
+my_permissions=我的权限
+all_permissions=所有权限
+key.categories.serverutilities=服务器实用工具
+stand_still=请在%d秒内保持站立不动。
+stand_still_failed=你移动了！传送已取消。
+teleporting=正在传送...
+cooldown=冷却时间
+cant_use_now=现在不能使用！
+cant_use_now_cooldown=现在不能使用！冷却时间：%s
+action_bar=操作栏
+nothing_changed=没有任何更改！
+
+serverutilities.task.cleanup_entity=服务器将在 %d 秒内清除所有 %s
```

Comment:
en_US.lang is (%s, %d), but here is (%d, %s). Will this cause some bug?

Reply:
![_XC J%Q(Y$_B54XVXFLA8TO](https://github.com/user-attachments/assets/aef2c843-900b-4cb9-bc74-294c06845cba)
``` txt
//en
serverutilities.task.cleanup_entity=Removing %s in %d seconds
//zh
serverutilities.task.cleanup_entity=服务器将在 %d 秒内清除所有 %s
```

Reply:
```suggestion
serverutilities.task.cleanup_entity=服务器将在 %2$d 秒内清除所有 %1$s
```